symplyphysics/laws/thermodynamics/heat_transfer_equation.py

```diff
@@ -0,0 +1,52 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output)
+
+# Description
+# Fourier's law Q = k * (T1-T2)/ d * S
+# where:
+# Q = heat flow 
+# k = material thermal conductivity, ability of a material to transfer heat
+# T1 = inside temperature
+# T2 = outside temperature
+# d = cross section thickness
+# S = cross-sectional area
+# 
+# The Fourier formula is used to determine the amount of heat flow that is transferred through a material from one side to the other due to the material's thermal conductivity, temperature gradient, thickness and cross-sectional area. 
+# The cross-sectional area can be any shape.
+# And the formula is suitable for calculating the heat flow between the external and internal temperatures of the ball, indicating the thickness of the cross section of the ball and its surface area.
```

Comment:
With ball I meant a solid ball. It does not have constant thickness. Is it possible to apply this law to this kind of obstacle?